scripts/buildZ3

```diff
@@ -0,0 +1,62 @@
+#!/bin/sh
+
+# Fail the whole script if any command fails
+set -e
+
+myDir="`dirname $0`"
+case `uname -s` in
+    CYGWIN*)
+      myDir=`cygpath -m $mydir`
+      ;;
+esac
+
+if [ "$myDir" = "" ];
+then
+    myDir="."
+fi
+
+UBUNTU_Z3="https://github.com/Z3Prover/z3/releases/download/z3-4.5.0/z3-4.5.0-x64-ubuntu-14.04.zip"
```

Comment:
should these be tied to specific OS versions?

Reply:
Sorry, I didn't understand this comment...

What do you mean by "tied to specific OS versions?"

Currently I hardcode these two released link for supporting download pre-compiled z3 in OS X and Linux. In line 23 I had a check on the underlying OS version, in order to download correct version of z3.

Could you clarify your suggestion?

Thanks!

Reply:
What I mean is whether we must use the 14.04 version / whether 14.04 works with other ubuntu versions? I use 16.04.

Reply:
Ah, I didn't test it for all sub-versions of Ubuntu. I would guess it will generally support all sub-versions as there were only one z3 release for Ubuntu.

Please let me know if you run into sub-versions problem.

Reply:
Will do in the future. Lets not hold back the PR due to OS versions. Please add a TODO note (for me) though.